tests/unit/backend/corpora/common/entities/test_project.py

```diff
@@ -23,10 +35,4 @@ def test__get__does_not_exist(self):
 
     def test__get__invalid_status(self):
         invalid_status_key = (self.uuid, "invalid_status")
-
```

Comment:
I think it's worth retaining the check to make sure the logging was successful - I do this in the DatasetValidator tests to make sure that the right error is logged.

Reply:
Will do

Reply:
fixed